mypy/fastparse.py

```diff
@@ -171,11 +171,21 @@ def ast3_parse(
     MatchAs = Any
     MatchOr = Any
     AstNode = Union[ast3.expr, ast3.stmt, ast3.ExceptHandler]
+
 if sys.version_info >= (3, 11):
     TryStar = ast3.TryStar
 else:
     TryStar = Any
 
+if sys.version_info >= (3, 12):
+    ast_TypeAlias = ast3.TypeAlias
+    ParamSpec = ast3.ParamSpec
+    TypeVarTuple = ast3.TypeVarTuple
+else:
+    ast_TypeAlias = Any
+    ParamSpec = Any
+    TypeVarTuple = Any
```

Comment:
Maybe use `ast_ParamSpec` and `ast_TypeVarTuple` here instead, just to prevent accidental name conflicts in the future?

Reply:
Done.